templates/workloads/jobs/scheduled-job/cf.yml

```diff
@@ -40,20 +40,8 @@ Resources:
     DependsOn: LogGroup
     Properties:
 {{include "fargate-taskdef-base-properties" . | indent 6}}
-      ContainerDefinitions:
-        - Name: !Ref WorkloadName
-          Image: !Ref ContainerImage
+{{include "container-definition-base-properties" . | indent 6}}
 {{include "envvars" . | indent 10}}
```

Comment:
Was there a reason why we didn't include "envvars" to the new template?

Reply:
LB web services have the LB URL as an extra environment variable and that adds messy conditionals.

Reply:
Fixed in latest commit.